tests/algolia/rules_engine_test.py

```diff
@@ -0,0 +1,110 @@
+from algolia.rules_engine import is_eligibilite_for_indexing
+from models import PcObject
+from tests.conftest import clean_database
+from tests.test_utils import create_offer_with_thing_product, create_offerer, create_stock, create_venue
+
+
+class IsEligibiliteForIndexingTest:
+    @clean_database
+    def test_should_return_false_when_none_is_given(self, app):
+        # Given
+        offerer = create_offerer()
+        venue = create_venue(offerer=offerer)
+        offer_in_base = create_offer_with_thing_product(venue=venue, idx=1)
+        stock = create_stock(offer=offer_in_base)
+        PcObject.save(stock)
+
+        # When
+        is_eligibilite = is_eligibilite_for_indexing(None)
```

Comment:
```suggestion
        is_eligible = is_eligible_for_indexing(None)
```
(même commentaire pour les prochains tests)